src/components/sideMenu/view/sideMenuView.js

```diff
@@ -20,53 +20,46 @@ import styles from './sideMenuStyles';
 // Images
 const SIDE_MENU_BACKGROUND = require('../../../assets/side_menu_background.png');
 
-class SideMenuView extends Component {
-  /* Props
-   * ------------------------------------------------
-   *   @prop { type }    name                - Description....
-   */
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      menuItems: props.isLoggedIn ? MENU.AUTH_MENU_ITEMS : MENU.NO_AUTH_MENU_ITEMS,
-      isAddAccountIconActive: false,
-      storageT: 'R',
-    };
-  }
+const SideMenuView = ({
+  currentAccount,
+  isLoggedIn,
+  intl,
```

Comment:
you can not get intl with this. you can use useIntl() hooks

Reply:
ok!